src/main/java/com/microsoft/sqlserver/jdbc/DataTypes.java

```diff
@@ -981,7 +997,8 @@ public int getIntValue() {
                 JDBCType.Category.CHARACTER,
                 JDBCType.Category.LONG_CHARACTER,
                 JDBCType.Category.NCHARACTER,
-                JDBCType.Category.LONG_NCHARACTER)),
+                JDBCType.Category.LONG_NCHARACTER,
+                JDBCType.Category.SQL_VARIANT)), 
```

Comment:
Also, both getter conversion and updater converstion have entries for SQL_Variant, should we also have it for setter conversion too?